llvm.yaml

```diff
@@ -0,0 +1,830 @@
+          mv ${{targets.destdir}}/${{vars.clang-prefix}}/share/*abilist* ${{targets.contextdir}}/${{vars.clang-prefix}}/share/
+          mv ${{targets.destdir}}/${{vars.clang-prefix}}/share/*ignorelist* ${{targets.contextdir}}/${{vars.clang-prefix}}/share/
+
+          mkdir -p ${{targets.contextdir}}/${{vars.clang-prefix}}/include
+          mv ${{targets.destdir}}/${{vars.clang-prefix}}/include/fuzzer ${{targets.contextdir}}/${{vars.clang-prefix}}/include/
+          mv ${{targets.destdir}}/${{vars.clang-prefix}}/include/orc ${{targets.contextdir}}/${{vars.clang-prefix}}/include/
+          mv ${{targets.destdir}}/${{vars.clang-prefix}}/include/profile ${{targets.contextdir}}/${{vars.clang-prefix}}/include/
+          mv ${{targets.destdir}}/${{vars.clang-prefix}}/include/sanitizer ${{targets.contextdir}}/${{vars.clang-prefix}}/include/
+          mv ${{targets.destdir}}/${{vars.clang-prefix}}/include/xray ${{targets.contextdir}}/${{vars.clang-prefix}}/include/
+
+  - name: clang-analyzer
+    description: Clang ${{vars.major-version}} analyzer
+    dependencies:
+      provides:
+        - clang-${{vars.major-version}}-analyzer=${{package.full-version}}
+      runtime:
+        - perl
+        - python3
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/${{vars.llvm-prefix}}/bin
+          mv ${{targets.destdir}}/${{vars.llvm-prefix}}/bin/scan* ${{targets.contextdir}}/${{vars.llvm-prefix}}/bin/
+          mv ${{targets.destdir}}/${{vars.llvm-prefix}}/bin/analyze-build* ${{targets.contextdir}}/${{vars.llvm-prefix}}/bin/
+          mv ${{targets.destdir}}/${{vars.llvm-prefix}}/bin/intercept-build* ${{targets.contextdir}}/${{vars.llvm-prefix}}/bin/
+
+          mkdir -p ${{targets.contextdir}}/usr/bin
+          for path in ${{targets.contextdir}}/${{vars.llvm-prefix}}/bin/*; do
+            name=${path##*/}
+            ln -s ../lib/llvm-${{vars.major-version}}/bin/$name ${{targets.contextdir}}/usr/bin/$name
+          done
+
+          mkdir -p ${{targets.contextdir}}/${{vars.llvm-prefix}}/libexec
+          mv ${{targets.destdir}}/${{vars.llvm-prefix}}/libexec/*-analyzer* ${{targets.contextdir}}/${{vars.llvm-prefix}}/libexec/
+          mv ${{targets.destdir}}/${{vars.llvm-prefix}}/libexec/analyze-* ${{targets.contextdir}}/${{vars.llvm-prefix}}/libexec/
+          mv ${{targets.destdir}}/${{vars.llvm-prefix}}/libexec/intercept-* ${{targets.contextdir}}/${{vars.llvm-prefix}}/libexec/
+
+          mkdir -p ${{targets.contextdir}}/${{vars.llvm-prefix}}/share
+          mv ${{targets.destdir}}/${{vars.llvm-prefix}}/share/scan-* ${{targets.contextdir}}/${{vars.llvm-prefix}}/share/
+
+          mkdir -p ${{targets.contextdir}}/${{vars.llvm-prefix}}/lib
+          mv ${{targets.destdir}}/${{vars.llvm-prefix}}/lib/libear* ${{targets.contextdir}}/${{vars.llvm-prefix}}/lib/
+    test:
+      pipeline:
+        - runs: |
+            scan-build --help
+            scan-view --help
```

Comment:
It looks like there are also binaries for `analyze-build` and `intercept-build` do those have arguments like `--help` or `--version` that we could test?